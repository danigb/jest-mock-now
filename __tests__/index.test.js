const index = require('../');

test('Date.now() to match snapshot w/o options', () => {
  const _ = 1479427200000;
  index();

  expect(Date.now()).toEqual(_);
  expect(Date.now()).toMatchSnapshot();
});

test('Date.now() to match snapshot w/ options', () => {
  const now = new Date('2017-06-22');
  index(now);

  expect(Date.now()).toEqual(now.getTime());
  expect(Date.now()).toMatchSnapshot();
});
