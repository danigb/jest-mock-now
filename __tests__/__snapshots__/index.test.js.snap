// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Date.now() to match snapshot w/ options 1`] = `1498089600000`;

exports[`Date.now() to match snapshot w/o options 1`] = `1479427200000`;
